services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java

```diff
@@ -181,7 +181,7 @@ protected void updateEligibilityCheck(Table existingTable, TableDto tableDto) {
    * Ensure existing table's tableLocation (path to metadata.json) matches user provided baseVersion
    * (path to metadata.json of the table where the updates are based upon)
    */
-  private void versionCheck(Table existingTable, TableDto mergedTableDto) {
+  void versionCheck(Table existingTable, TableDto mergedTableDto) {
```

Comment:
Is this change to support extensibility?

Reply:
yes exactly the purpose.